task-4/task-4.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+func bo(a [][]int, n int) bool {
+	c := 0
+	for i := 0; i < n; i++ {
+		for j := 0; j < n; j++ {
+			if a[i][j] == a[j][i] {
+				c++
+			}
+		}
+	}
+	if c == n*n {
+		return true
+	} else {
+		return false
+	}
```

Comment:
смотри, когда используешь такие конструкции, можно писать лаконичнее и изящнее
```suggestion
	if c == n*n {
		return true
	}
	return false
```
```else```  не нужен, тк ```return false``` не выполнится, если выполнится ```return true``` (функция завершит своё исполнение)

Reply:
ИСПРАВЛЕНО